scripts/newinstall.sh

```diff
@@ -188,69 +206,69 @@ if true; then
 		echo " option to point to system Python 2 interpreter and rerun."
 		exit -1;
 	fi
-
+	
 	if [[ "$PYTHON" != "/usr/bin/python" ]]; then
 		echo "Using python at $PYTHON to install EUPS"
 	fi
-
+	
 	if [[ -z $EUPS_GITREV ]]; then
 		echo -n "Installing EUPS (v$EUPS_VERSION)... "
 	else
 		echo -n "Installing EUPS (branch $EUPS_GITREV from $EUPS_GITREPO)..."
 	fi
-
+	
 	(
-		mkdir _build && cd _build
-		if [[ -z $EUPS_GITREV ]]; then
-			# Download tarball from github
-			curl -L $EUPS_TARURL | tar xzvf -
-			cd eups-$EUPS_VERSION
-		else
-			# Clone from git repository
-			git clone "$EUPS_GITREPO"
-			cd eups
-			git checkout $EUPS_GITREV
-		fi
-
-		./configure --prefix="$LSST_HOME"/eups --with-eups="$LSST_HOME" --with-python="$PYTHON"
-		make install
-
+	mkdir _build && cd _build
```

Comment:
I think you want to indent here to make it very clear all of this gets executed in a subshell (the parentheses)